app/common/lib/platformUtil.js

```diff
@@ -42,3 +42,8 @@ module.exports.isWindows = () => {
   return process.platform === 'win32' ||
     navigator.platform === 'Win32'
 }
+
+module.exports.isLinux = () => {
+  return !module.exports.isDarwin() &&
+    !module.exports.isWindows()
+}
```

Comment:
this is perfect! For anyone else looking at this... I believe this is the best way because there could be many variations of `navigator.platform` for Linux. We can't rely on process.platform because this code is often invoked in the renderer (like in this PR)